bin/server.js

```diff
@@ -159,13 +159,21 @@ server.post('/conversation', async (request, reply) => {
 server.listen({
     port: settings.apiOptions?.port || settings.port || 3000,
     host: settings.apiOptions?.host || 'localhost',
-}, (error) => {
+    }, (error, address) => {
     if (error) {
         console.error(error);
         process.exit(1);
     }
+    console.log(`Server is listening on ${address}`)
 });
 
+// // wait for the server to start
+// await new Promise(resolve => server.ready(resolve));
+
+// fastify.on('listening', (address) => {
+//     console.log(`Server listening on ${address}`);
+// });
+
```

Comment:
You should delete code rather than comment it out, since they can be restored via git if still needed in the future.

Reply:
so~what should i do~?